framework/wazuh/security_configuration_assessment.py

```diff
@@ -65,9 +65,6 @@ def __init__(self, agent_id, offset, limit, sort, search, select, query, count,
     def _default_query(self):
         return self.default_query
 
-    def _default_count_query(self):
-        return f"COUNT(DISTINCT {self.count_field})"
-
```

Comment:
We should preserve this function but changing the count query as the super class does.

Reply:
Added:
`return f"SELECT COUNT(DISTINCT {self.count_field})" + " FROM ({0})"`